### DISCUSSION TITLE
SpellQueuewindow

### DISCUSSION THREAD

**Discussion by Reily93** [chromiecraft/chromiecraft#7621] [Suggestions]
I saw this fix upcoming and i would like to give  a suggestion,https://github.com/azerothcore/azerothcore-wotlk/commit/993bdcb84eaa8eeda92d8803a208311dfa831ce1

I saw that the window is setted to 400 as the standard blizzard of <2010 year , so the connection at time were really bad , so considering the differences in latency of nowdays i would like make a poll of a average cc players latency, on that set the server latency to that average value eu+na + 100 , why 100 cause on every player experience u set ur latency + 100 asn vbest optimal queuewindow 

i know is not AC goal but implementing single player pèersonal queue window would be cool and nice to kill that spell batching , even tho is not a woltk feature is a nice feature introduced with cataclysm whis is not a hame content but a system improvement to counter lag.

make a poll of ppl who have :

1. <100 latency 
2. 100-200
3.  more than 200

and set the average result + 100 cause using the old 400 is a mistake cause nowdays we have better internet connections

**Comment by Reily93** [comment]
the latency stup is too high there is too much deley pls decrease it

**Reply by Helias** [reply]
do you have any official sources about which is the blizzlike one for wotlk 3.3.5a?

**Reply by Reily93** [reply]
Hi Helias thx for reaching,  by checking on the internet the value that i found is 400 ms, on the 3.3.5 woltk, later even in classic 3.4.0 that value can be setted to  personal choice so everyone can put his own lag tolerance, and in a context like 2006-2010,400 ms was a nice value nowdays it isnt ,i personally experience too much delay, i would rather set a value to half of it 200 or develop personal queue window for each player, and wpuld be cool to be the first 3.3.5 server to have that honestly. Now i dont have knowledge how hard is to do . if is possible , and if take too much time , so i would rather decrease the default to a value more appropiate for  2024 i guess 200 ms latency default is a nice compromise

**Comment by Nyeriah** [comment]
Lowered to 100ms for the time being, definitive solution to come later

**Reply by armchairrockstar1** [reply]
I think this was the right call. Thank you.
